pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -368,12 +368,50 @@ func startDnsmasq(bridgeName string) {
 	}
 }
 
-//    pkill -u nobody -f dnsmasq.${BRIDGENAME}.conf
+//    kill -9 (-0) pid of dnsmasq on bridge
 func stopDnsmasq(bridgeName string, printOnError bool, delConfiglet bool) {
 
 	log.Infof("stopDnsmasq(%s)\n", bridgeName)
-	cfgFilename := dnsmasqConfigFile(bridgeName)
-	pkillUserArgs("root", cfgFilename, printOnError)
+	pidfile := fmt.Sprintf("/var/run/dnsmasq.%s.pid", bridgeName)
+	pid, err := ioutil.ReadFile(pidfile)
+	if err != nil {
+		log.Errorf("stopDnsmasq: pid file read error %v\n", err)
+		return
+	}
+	pidStr := string(pid)
+	pidStr = strings.TrimSuffix(pidStr, "\n")
+
+	cmd := exec.Command("kill", "-9", pidStr)
```

Comment:
No need to use exec for this. os.FindProcess() and the Signal as in dhcpcd.go is better